opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -306,101 +299,6 @@ namespace Opm
             ThreadManager::init();
         }
 
-        // Extract the minimum priority and determines if log files ought to be created.
-        // Writes to:
-        //   output_to_files_
-        //   output_
-        void setupOutput()
-        {
-            const std::string outputModeString =
-                EWOMS_GET_PARAM(TypeTag, std::string, OutputMode);
-            static std::map<std::string, FileOutputMode> stringToOutputMode =
-                { {"none", OUTPUT_NONE },
-                  {"false", OUTPUT_LOG_ONLY },
-                  {"log", OUTPUT_LOG_ONLY },
-                  {"all" , OUTPUT_ALL },
-                  {"true" , OUTPUT_ALL }};
-            auto outputModeIt = stringToOutputMode.find(outputModeString);
-            if (outputModeIt != stringToOutputMode.end()) {
-                output_ = outputModeIt->second;
-            }
-            else {
-                output_ = OUTPUT_ALL;
-                std::cerr << "Value " << outputModeString <<
-                    " is not a recognized output mode. Using \"all\" instead."
-                          << std::endl;
-            }
-
-            output_cout_ = false;
```

Comment:
The `output_cout_` and `output_to_files_` variables must still be initialized properly in this class.

Reply:
Yes - and that inspired a quite significant reworking of this PR!